batch/itests/org.jboss.tools.batch.core.itest/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,42 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Batch Core Test
+Bundle-SymbolicName: org.jboss.tools.batch.core.itest
+Bundle-Version: 1.7.0.qualifier
+Bundle-Vendor: JBoss by Red Hat
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Require-Bundle: org.junit,
+ org.jboss.tools.tests,
+ org.jboss.tools.batch.core,
+ org.jboss.tools.common.base.test,
+ org.eclipse.core.resources,
+ org.jboss.tools.common,
+ org.eclipse.core.runtime,
+ org.eclipse.ui,
+ org.eclipse.jface,
+ org.eclipse.jface.text,
+ org.eclipse.jdt.core,
+ org.eclipse.jdt.ui,
+ org.eclipse.ui.ide,
+ org.jboss.tools.jst.jsp.base.test,
+ org.eclipse.ltk.core.refactoring,
+ org.eclipse.search,
+ org.jboss.tools.common.el.core,
+ org.jboss.tools.jst.web.kb,
```

Comment:
Well, KB is more then web actually. But historically the KB plugin has a 'web' in it's ID.